src/unitxt/loaders.py

```diff
@@ -19,6 +20,13 @@
 
 
 class Loader(SourceOperator):
+    # The loader_limit an optional parameter used to control the maximum number of instances to load from the the source.
+    # It is usually provided to the loader via the recipe (see standard.py)
+    # The loader can use this value to limit the amount of data downloaded from the source
+    # to reduce loading time.  However, this may not always be possible, so the
+    # loader may ingore this.  In any case, the recipe, will limit the number of instances in the returned
```

Comment:
is the loader per card?
if so, isn't it possible to error out if its not possible to limit the load?
because the source is fixed in the card
Why not to throw an error?